cmd/testwrapper/testwrapper.go

```diff
@@ -273,6 +278,17 @@ func main() {
 				}
 			}
 		}
+		if err := <-runErr; err != nil {
+			var exit *exec.ExitError
+			if errors.As(err, &exit) {
+				if code := exit.ExitCode(); code > -1 {
+					os.Exit(exit.ExitCode())
```

Comment:
This is in the wrong spot. We'll always get a nonzero exit code when there are flakey tests we want to retry.

Reply:
I've got this fixed but I think I'm going to write a bunch more test cases.

Reply:
Updated in https://github.com/tailscale/tailscale/pull/9276/commits/e1770713d58e44ad3947471d6071bbae78bcfc31